commit 232c80cfbcb47ea91f9189f049775cd69371a823
Author: Eric Anholt <eric@anholt.net>
Date:   Thu Oct 20 16:48:12 2016 -0700

    drm/vc4: Fix termination of the initial scan for branch targets.
    
    The loop is scanning until the original max_ip (size of the BO), but
    we want to not examine any code after the PROG_END's delay slots.
    There was a block trying to do that, except that we had some early
    continue statements if the signal wasn't a PROG_END or a BRANCH.
    
    The failure mode would be that a valid shader is rejected because some
    undefined memory after the PROG_END slots is parsed as a branch and
    the rest of its setup is illegal.  I haven't seen this in the wild,
    but valgrind was complaining when about this up in the userland
    simulator mode.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>
    (cherry picked from commit 457e67a728696c4f8e6423c64e93def50530db9a)

Index: linuxtest/dummy/rpi_1161_232c80cfbcb47ea91f9189f049775cd69371a823.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1161_232c80cfbcb47ea91f9189f049775cd69371a823.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
